app/_includes/head.html

```diff
@@ -49,7 +49,6 @@
   <link rel="dns-prefetch" href="//cdn.segment.com">
   <link rel="dns-prefetch" href="//sidecar.gitter.im">
   <link rel="dns-prefetch" href="//buttons.github.io">
-  ##preload_assets##
```

Comment:
@thefosk that token should be replaced by a string on `npm start` https://github.com/Mashape/getkong.org/blob/master/gulpfile.js#L153

You shouldn't remove it. Was it showing up after gulp?